pyproject.toml

```diff
@@ -1,54 +1,61 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools>=63.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools.packages.find]
 include = ["sopel_unicode", "sopel_unicode.*"]
 namespaces = false
 
 [tool.setuptools.dynamic]
-readme = { file=["README.rst", "NEWS"], content-type="text/x-rst" }
+readme = { file=["README.md", "NEWS"], content-type="text/markdown" }
 
 [project]
 name = "sopel-unicode"
 version = "1.0.0"
 description = "Unicode information lookup plugin for Sopel"
+
+authors = [
+  { name="SnoopJ" },
+]
+
+license = "EFL-2.0"
+license-files = ["COPYING"]
+dynamic = ["readme"]
+
+classifiers = [
+  "Intended Audience :: Developers",
+  "Intended Audience :: System Administrators",
+  "Topic :: Communications :: Chat :: Internet Relay Chat",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+]
 keywords = [
   "sopel",
   "plugin",
   "bot",
   "irc",
 ]
-maintainers = [
-  { name="SnoopJ", email="snoopjedi@gmail.com" },
-]
-authors = [
-  { name="SnoopJ", email="snoopjedi@gmail.com" },
+
+requires-python = ">=3.8, <4"
+dependencies = [
+  "sopel>=8.0",
 ]
 
-dynamic = ["readme"]
-license = { text="EFL-2.0" }
-classifiers = [
-    "Intended Audience :: Developers",
-    "Intended Audience :: System Administrators",
-    "License :: Eiffel Forum License (EFL)",
-    "License :: OSI Approved :: Eiffel Forum License",
-    "Operating System :: POSIX :: Linux",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Programming Language :: Python :: 3.13",
-    "Topic :: Communications :: Chat :: Internet Relay Chat",
+[project.optional-dependencies]
+UCD = ["unicodedata2"]
+age = ["unicode_age"]
+all = [
+  "sopel-unicode[UCD]",
+  "sopel-unicode[age]",
 ]
-requires-python = ">=3.8, <4"
-dependencies = ["sopel>=7.1"]
 
 [project.urls]
-"Homepage" = "https://github.com/snoopj/sopel-unicode"
-"Bug Tracker" = "https://github.com/snoopj/sopel-unicode/issues"
+"Homepage" = "https://github.com/sopel-irc/sopel-unicode"
+"Bug Tracker" = "https://github.com/sopel-irc/sopel-unicode/issues"
 
 [project.entry-points."sopel.plugins"]
-unicode_info = "sopel_unicode"
-unicode_extras = "sopel_unicode.extras"
+"sopel-unicode" = "sopel_unicode.plugin"
```

Comment:
I'm pretty sure this should be either `unicode_info` (to "replace" the old built-in) or just `unicode`.

Reply:
I do want to keep the two plugins distinct, but I need to add a section on migration to the `README` mentioning that the built-in should be disabled when installing this plugin.